templates/hardware.html

```diff
@@ -0,0 +1,166 @@
+{% extends '_layout.html' %} {% block title%} Ubuntu Desktop certified hardware
+{% endblock %} {% block content %}
+
+<nav role="navigation" class="nav-secondary clearfix">
+  <ul class="breadcrumb">
+    <li>
+      <a href="/desktop/">
+        Ubuntu Desktop
+      </a>
+    </li>
+    <li>&nbsp;›</li>
+
+    <li>
+      <a href="https://certification.ubuntu.com/desktop/models/?category=Desktop&amp;category=Laptop&amp;vendors=Dell">
```

Comment:
I noticed that on https://certification.ubuntu.com/hardware/201612-25299/, the "Search results" button takes you back to `/desktop/models/?vendors=Dell`, whereas you have it going back to `/desktop/models/?category=Desktop&category=Laptop&vendors=Dell`.

Was this an intentional change? Unless there's a reason, I think we should strive to keep it the same as on live.

Reply:
ah nice

Reply:
So I guess this should go back to the previous page in general. Maybe this needs to be implemented on another level and then used as a partial in the templates?